src/App.jsx

```diff
@@ -1,8 +1,35 @@
 import React from 'react';
+import { Link, Outlet, RouterProvider, createMemoryRouter } from 'react-router-dom';
 import './assets/styles/styles.scss';
+import Modal from './components/Modal';
+
+const router = createMemoryRouter(
```

Comment:
👍

Reply:
I am using a memory router because it seems to be the only option to work when using the app as a browser extension.

Reply:
Yeah, that is totally fine. Good choice.